.CI/create_feedstocks.py

```diff
@@ -126,6 +157,19 @@ def write_token(name, token):
             out = subprocess.check_output(['git', 'push', 'upstream_with_token', 'master'], cwd=feedstock_dir,
                                           stderr=subprocess.STDOUT)
 
+            # Add team members as maintainers.
+            if conda_forge:
+                meta = MetaData(recipe_dir)
+                maintainers = meta.meta.get('extra', {}).get('recipe-maintainers', [])
+                team = create_team(
```

Comment:
What happens if we try to add a team twice? Do we have to handle some error? Does it just give us the existing team?